sdk/src/main/java/co/omise/android/models/Bank.kt

```diff
@@ -0,0 +1,11 @@
+package co.omise.android.models
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class Bank(
+        var name: String? = null,
+        var code: String? = null,
+        var active: Boolean? = false,
+) : Parcelable {}
```

Comment:
```suggestion
) : Parcelable
```
Could you remove empty body?